storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1445,9 +1464,10 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
   archived_log_monitor.stop();
 
   /* to ensure redo logs are not disabled during the backup, reopen the log
-  files to read HEADER */
-  if (!reopen_log_files(lsn)) {
-    xb::error() << "Cannot find file with LSN " << lsn;
+  files to read HEADER. */
+  if (opt_lock_ddl == false && archived_log_state == ARCHIVED_LOG_NONE &&
+      !reopen_log_files()) {
+    xb::error() << "Failed to open redo log files. ";
```

Comment:
if this call is to ensure redo logs have not been disabled, can we add some hint on the log message?

```
    xb::error() << "Failed to open redo log files. Are redo_logs disabled?";
```

Reply:
I also though to add but that would be duplicate. Because if redo is disabled, `log_files_find_and_analyze` will add a message about that.

Reply:
Can you show the error message?

Reply:
`2023-02-06T11:21:52.050353+05:30 0 [ERROR] [MY-013598] [InnoDB] Redo logging is disabled, cannot take consistent backup 2023-02-06T11:21:52.050366+05:30 0 [ERROR] [MY-011825] [Xtrabackup] Failed to open redo log files. 2023-02-06T11:21:52.050373+05:30 0 [ERROR] [MY-011825] [Xtrabackup] Error stopping copy thread at LSN 20619739`

Reply:
@satya-bodapati  yes we do grep for "Redo logging is disabled, cannot take consistent backup" inside test case `alter_instance_redo.sh`